src/components/NavBar/NavigationBar.tsx

```diff
@@ -0,0 +1,188 @@
+        display: flex;
+    }
+`;
+
+const Spacer = styled.div`
+    flex-grow: 1;
+`;
+
+const DrawerInner = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: start;
+    align-items: stretch;
+    padding: 1rem;
+    width: 15rem;
+`;
+
+const LogoNavElement: React.FC<{}> = () => (
+    <UnstyledLink href="/">
+        <LogoDiv>
+            <Logo alt="Vauxhall Foodbank Logo" src="/logo.webp" />
+        </LogoDiv>
+    </UnstyledLink>
+);
+
+const Gap = styled.div`
+    width: 1rem;
+`;
+
+const DrawerButtonWrapper = styled.div`
+    padding: 1rem;
+    border-bottom: 1px solid ${(props) => props.theme.secondaryBackgroundColor};
+
+    &:last-child {
+        border-bottom: none;
+    }
+`;
+
+const DrawerButton = styled(Button)`
+    width: 100%;
+    color: ${(props) => props.theme.secondaryBackgroundColor};
+`;
+
+const StickyAppBar = styled(AppBar)`
+    position: sticky;
+    `;
+
+const ResponsiveAppBar: React.FC<{}> = () => {
+    const [drawer, setDrawer] = React.useState(false);
+
+    const openDrawer = (): void => {
+        setDrawer(true);
+    };
+
+    const closeDrawer = (): void => {
+        setDrawer(false);
+    };
+
+    const pages = [
+        ["Clients", "/clients"],
+        ["Lists", "/lists"],
+        ["Calendar", "/calendar"],
+    ];
+
+    const setThemeMode = useContext(ThemeUpdateContext);
+
+    const switchThemeMode = (event: React.ChangeEvent<HTMLInputElement>): void => {
+        setThemeMode(event.target.checked);
+    };
+
+    return (
+        <>
+            <SwipeableDrawer open={drawer} onClose={closeDrawer} onOpen={openDrawer}>
+                <DrawerInner>
+                    {pages.map(([page, link]) => (
+                        <DrawerButtonWrapper key={page}>
+                            <UnstyledLink href={link} onClick={closeDrawer}>
+                                <DrawerButton variant="text">{page}</DrawerButton>
+                            </UnstyledLink>
+                        </DrawerButtonWrapper>
+                    ))}
+                </DrawerInner>
+            </SwipeableDrawer>
+            <StickyAppBar position="static">
+                <AppBarInner>
+                    <SmallNavBarElement>
+                        <IconButton
+                            size="medium"
+                            aria-label="Mobile Menu Button"
+                            onClick={openDrawer}
+                        >
+                            <MenuIcon />
+                        </IconButton>
+                        <Spacer />
+                    </SmallNavBarElement>
+                    <LogoNavElement />
+                    {/* TODO: VFB-16 need to fix nav colours */}
+                    <LgNavElement>
+                        {pages.map(([page, link]) => (
+                            <UnstyledLink key={page} href={link} onClick={closeDrawer}>
+                                <Button variant="outlined" color="secondary">
```

Comment:
Instead of UnstyledLink and Button separately, can you make a new component that does this in the components folder? (Potentially as `Link`?)
It's just if we want to change the styling of all buttons it's much easier to do it once in the components folder than it is CTRL-F'ing through code.
(Also, what does an MUI button have better than just a Link?)

Reply:
Not sure it's necessarily better? Just because throughout we use `UnstyledLink` around other things such as the Logo and the Drawer Buttons, which each require different styling, so if we want to change the styling we'd have to individually change all of those anyway. The MUI button provides a nice `onHover` animation.